tests/parsing/GlobbingTest.py

```diff
@@ -100,6 +101,41 @@ def test_alternatives(self):
                              sorted(alternatives))
 
 
+class EscapingTest(unittest.TestCase):
```

Comment:
imo it's better if you call the test class `EscapeGlobTest` so the name resembles the function better^^